src/components/Tooltip/Tooltip.js

```diff
@@ -1,135 +1,169 @@
-import React from 'react';
 import PropTypes from 'prop-types';
 import styled, { css } from 'react-emotion';
+import { includes } from 'lodash';
 
-import { textKilo } from '../../styles/style-helpers';
+import {
+  textKilo,
+  centerAlign,
+  shadowSingle
+} from '../../styles/style-helpers';
 
 const CENTER = 'center';
+const TOP = 'top';
 const RIGHT = 'right';
+const BOTTOM = 'bottom';
 const LEFT = 'left';
 
-const baseStyles = css`
+const backgroundColor = ({ theme }) => theme.colors.n900;
```

Comment:
Why does this need to be a function? Should be fine to use it directly, no?

Reply:
`theme` would be undefined.